public/angular/templates/incidents/index.html

```diff
@@ -21,7 +21,7 @@
         </div>
       </div>
     </div>
-    <div class="filter">
+    <div ng-hide="currentUser.role === 'viewer'" class="filter">
```

Comment:
All these updates to the UI seem great, but can we attach it to permissions instead of roles?